package.json

```diff
@@ -2,21 +2,22 @@
   "name": "@edx/paragon",
   "version": "0.0.0-development",
   "description": "Accessible, responsive UI component library based on Bootstrap.",
-  "main": "themeable/index.js",
+  "main": "dist/themeable/index.js",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/edx/paragon.git"
   },
   "scripts": {
-    "build": "NODE_ENV=production webpack --config webpack.config.js",
+    "build": "webpack --mode production",
     "build-storybook": "build-storybook",
+    "commit": "commit",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand --coverage",
     "deploy-storybook": "storybook-to-ghpages",
     "deploy-storybook-ci": "storybook-to-ghpages --ci",
     "gc": "commit",
```

Comment:
can we get rid of this alternative alias? This name I find very counterintuitive

Reply:
removed.

Reply:
Sorry I was unclear about which line I was pointing to in github's interface; it's much clear when adding a comment

Your name makes loads more sense to me than `npm run gc`. It's a pretty bad false-cognate with `git gc`

Reply:
Got it, changed.